CancerGov/_src/Scripts/NCI/Modules/bestBets/bestBets.js

```diff
@@ -0,0 +1,156 @@
+	var _fetchTerm = function () {
+		var searchParam = $('.results .term:first').text();
+		return searchParam;
+	};
+
+	// get the full definition from the dictionary service
+	var _getDefinition = function (term) {
+		return DictionaryService.search('term', _fetchTerm(), language);
+	};
+
+	// render the defintion to produce the content and html
+	var _render = function (obj) {
+		console.log("inside render with:", obj);
+		var term = '<dt>' + obj.term + '</dt>';
+		var audio = "";
+		var pronunciation = "";
+		
+		// check if pronunciation exists before building the audio and pronunciation of the definition
+		if (obj.pronunciation) {
+			audio = '<a href="' + obj.pronunciation.audio + '" class="CDR_audiofile"><span class="hidden">listen</span></a> ';
+			pronunciation = obj.pronunciation.key;
+		}
+		// toggle controls whether to show first sentence or full definition in mobile
+		var toggle = '<div id="best-bets-toggle"><a href="#"><span id="definition-arrow"></span><span id="definitionShowHide">' + config.lang.Show[lang] + '</span> ' + config.lang.Definition_Show_Full[lang] + '</a></div>';
+	
+		var definition = obj.definition.html;
+		// break the definition into first sentence and rest for display in mobile
+		var definitionStart =  obj.definition.html.slice(0, definition.indexOf('. ') + 1);
+		var definitionEnd = obj.definition.html.slice(definition.indexOf('. ') + 1);
+		
+		// in cases where there is only one sentence, assign it to first and make the rest an empty string
+		if (definitionStart === '') {
+			definitionStart = '<span id="definitionStart">' + definition + '</span>';
+			definitionEnd = '';
+			toggle = '';
+		}
+		else {
+			definitionStart = '<span id="definitionStart">' + definitionStart + '</span>';
+			definitionEnd = '<span id="definitionEnd">' + definitionEnd + '</span>';
+		}
+		
+
+		console.log("definitionStart is:" + definitionStart);
+		console.log("definitionEnd is:" + definitionEnd);
+		
+		// the box takes all the components and puts them together in the defintion box
+		var box = '<div id="best-bet-definition"><h2>' + config.lang.Definition_Title[lang] + ':</h2><dl>' + term + '<dd>' + audio + pronunciation + ' </dd><dd>' + definitionStart + definitionEnd + moreInfo() + '</dd></dl>'  + toggle + ' </div><div id="dictionary_jPlayer"></div>';
+		
+		// html rules for within best bets
+		if ($('.featured.sitewide-results')[0]) {
+			var wrapper = $('<div class="large-4 small-12 columns collapse">' + box + '</div>');
+			var target = $('.featured.sitewide-results');
+			target.wrapInner('<div class="large-8 small-12 columns collapse"></div>').append(wrapper);
+		// html rules for not within best bets
+		} else {
+			var wrapper = $('<div>' + box + '</div>');
+			var target = $('.sitewide-results');
+			wrapper.insertBefore(target);
+		}
+		// functionality for changing from compact to full definition in mobile
+		var areWeExpanded = "yes";
+		$("#best-bets-toggle a").on("click", function (event) {
```

Comment:
Animation is janky when definition is 100% wide because text is inline element. Perhaps, render the text and then animate the parent.